pymeasure/adapters/ni_gpib_232ct.py

```diff
@@ -0,0 +1,346 @@
+            (without termination).
+        :param kwargs: Keyword arguments for the connection itself.
+        """
+        self.flush_read_buffer()
+        self.connection.write(f"wrt {self.address} \n  {command}", **kwargs)
+        time.sleep(0.050)
+        self._check_errors()
+        self.flush_read_buffer()
+
+    def _write_bytes(self, content, **kwargs):
+        """Write byte to the instrument appending `write_termination`.
+
+        :param str content: string to be sent to the instrument (without termination).
+        :param kwargs: Keyword arguments for the connection itself.
+        """
+        self.flush_read_buffer()
+        self.connection.write(f"wrt {self.address} \n  {content}", **kwargs)
+        time.sleep(0.050)
+        self._check_errors()
+        self.flush_read_buffer()
+
+    def assert_trigger(self):
+        """
+        Initiate a GPIB trigger-event
+        """
+        self.connection.write(f"trg {self.address}")
+
+    def clear(self):
+        """
+        Clear specified device.
+
+        """
+        self.connection.write(f"clr  {self.address}")
+
+    @property
+    def eoi(self):
+        """Control whether to assert the EOI signal with the last character
+        of any command sent over GPIB port (bool).
+
+        Some instruments require EOI signal to be
+        asserted in order to properly detect the end of a command.
+        """
+        self.connection.write("EOT")
+        return bool(int(self.connection.read()))
+
+    @eoi.setter
+    def eoi(self, value):
+        self.connection.write(f"EOT {int(value)}")
+
+    def gpib(self, address, **kwargs):
+        """Return a NI_GPIB_232 object that references the GPIB
+        address specified, while sharing the Serial connection with other
+        calls of this function
+
+        :param address: Integer GPIB address of the desired instrument
+        :param kwargs: Arguments for the initialization
+        :returns: NI_GPIB_232 for specific GPIB address
+        """
+        return NI_GPIB_232(self, address, **kwargs)
+
+    def pass_control(self, primary_address: int, secondary_address: int):
+        """
+        Pass control to drevice with primary_address and optional secondary_address
+
+        """
+        self.connection.write(f"pct  {primary_address}+{secondary_address}")
+
+    def send_command(self, data: bytes):
+        """
+        Write GPIB command bytes on the bus.
+
+        """
+        self.connection.write(f"cmd  #{len(data)}\n {data}")
+        self._check_errors()
+
+    def set_rsc(self):
+        """
+        set the NI-GPIB232ct to become the GPIB system controller
+
+        """
+        self.connection.write("rsc  1")
+
+    def send_ifc(self):
+        """Pulse the interface clear line (IFC) for at least 200 microseconds."""
+        self.connection.write("sic 0.0002")
+
+    @property
+    def time_out(self):
+        """Control control the GPIB timeout.
```

Comment:
```suggestion
        """Control the GPIB timeout.
```

I think, it is normally (VISA & Co.) called "timeout", not "time_out".

Reply:
Will change to the more common `timeout` and adjust the docstring as proposed